index.js

```diff
@@ -11,93 +11,128 @@ const githubClient = new GithubClient(token);
-    const alertsSummary = [];
-    if (criticalAlerts.length > 0) {
-        alertsSummary.push(`${criticalAlerts.length} critical`);
-    }
-    if (highAlerts.length > 0) {
-        alertsSummary.push(`${highAlerts.length} high`);
-    }
+  const alertsSummary = [];
+  if (criticalAlerts.length > 0) {
+    alertsSummary.push(`${criticalAlerts.length} critical`);
+  }
+  if (highAlerts.length > 0) {
+    alertsSummary.push(`${highAlerts.length} high`);
+  }
 
-    const blocks = [{
-        type: 'section',
-        text: {
-            type: 'mrkdwn',
-            text: `*<https://github.com/${org}/${name}|${org}/${name}>*\n${alertsSummary.join(', ')}\n<https://github.com/${org}/${name}/network/alerts|View all>`
-        },
-        accessory: {
-            type: 'image',
-            image_url: 'https://user-images.githubusercontent.com/10406825/85333522-ba846b80-b4a7-11ea-9774-46fa8ca693a4.png',
-            alt_text: 'github'
-        }
-    }];
-    criticalAlerts.forEach((criticalAlert) => {
-        blocks.push(formatAlertForSlack(criticalAlert));
-    });
-    highAlerts.forEach((highAlert) => {
-        blocks.push(formatAlertForSlack(highAlert));
-    });
+  const blocks = [
+    {
+      type: "section",
+      text: {
+        type: "mrkdwn",
+        text: `*<https://github.com/${org}/${name}|${org}/${name}>*\n${alertsSummary.join(
+          ", "
+        )}\n<https://github.com/${org}/${name}/network/alerts|View all>`,
+      },
+      accessory: {
+        type: "image",
+        image_url:
+          "https://user-images.githubusercontent.com/10406825/85333522-ba846b80-b4a7-11ea-9774-46fa8ca693a4.png",
+        alt_text: "github",
+      },
+    },
+  ];
+  criticalAlerts.forEach((criticalAlert) => {
+    blocks.push(formatAlertForSlack(criticalAlert));
+  });
+  highAlerts.forEach((highAlert) => {
+    blocks.push(formatAlertForSlack(highAlert));
+  });
 
-    return blocks;
+  return blocks;
 }
 
 function formatAlertForSlack({ id, package_name, severity, created_at }) {
-    return {
-        type: 'section',
-        block_id: `section-${id}`,
-        fields: [
-            {
-                type: 'mrkdwn',
-                text: `*Package (Severity Level)*\n${package_name} (${severity})`
-            },
-            {
-                type: 'mrkdwn',
-                text: `*Created on*\n${created_at}`
-            }
-        ]
-    }
+  return {
+    type: "section",
+    block_id: `section-${id}`,
+    fields: [
+      {
+        type: "mrkdwn",
+        text: `*Package (Severity Level)*\n${package_name} (${severity})`,
+      },
+      {
+        type: "mrkdwn",
+        text: `*Created on*\n${created_at}`,
+      },
+    ],
+  };
+}
+
+/**
+ * Slack only allows 50 blocks per message, break array of all blocks into groups of at most 50.
+ * Returns an array of arrays of blocks in order.
+ * @param {*} blocks
+ */
+function breakBlocks(blocks) {
```

Comment:
this function is the other change I made to break the whole list of blocks into smaller arrays of 50 or less. It returns an array of arrays of blocks.